packages/v4/patternfly-docs/content/accessibility/card/card.md

```diff
@@ -3,21 +3,109 @@ id: Card
+  </ListItem>
+</List>
+
+## React customization
+
+Various React props have been provided for more fine-tuned control over accessibility.
+
+| Prop | Applied to | Reason | 
+|---|---|---|
+| `aria-label` or `aria-labelledby` | `Card` | Adds an accessible name to the card. If the card has a `CardTitle`, the `aria-labelledby` prop should be passed in with a value of the `CardTitle`'s `id` prop. Otherwise the `aria-label` prop should be passed in to act as a title for the card. |
+| `component` | `Card` | Sets the base element for the sub-component (`article` element by default). If you primarily want to use a card for styling or layout purposes, it's recommended to pass in a value of "div" to this prop. |
+| `hasSelectableInput` | `Card` | Adds a visually hidden input that notifies users navigating via assistive technologies that a card is selectable. This prop should only be passed in when only one card within a set of cards should be selectable at a time, such as for a [primary-detail view](/demos/primary-detail). <br/><br/> When this prop is passed in, either a `CardTitle` sub-component must exist in the card, or the `selectableInputAriaLabel` prop must be passed in. The hidden input will automatically be given an accessible name when a `CardTitle` exists within the card, otherwise the value given to the `selectableInputAriaLabel` prop will add an accessible name to the input. |
+| `isDisabledRaised` | `Card` | Adds styling to visually indicate that a selectable card is disabled. When the `hasSelectableInput` prop is passed in, this prop will also set the selectable input's `disabled` attribute. |
+| `isExpanded` | `Card` | Adds styling to a card toggle to visually indicate whether the expandable content is expanded or collapsed. When this prop is passed in, `onExpand` and `toggleButtonsProps` must be passed into the `CardHeader` sub-component. |
+| `isSelectable` or `isSelectableRaised` | `Card` | Adds styling on hover or focus to visually indicate that a card is selectable, and adds the `tabindex="0"` attribute so that the card can be navigated to with standard keyboard navigation. |
+| `isSelected` | `Card` | Adds styling to visually indicate that a card is currently selected. When the `hasSelectableInput` prop is passed in, this prop will also set the selectable input's `checked` attribute. |
+| `selectableInputAriaLabel` | `Card` | Adds an accessible name to the hidden input when `hasSelectableInput` is passed in and there is no `CardTitle` sub-component within the card. The value passed into this prop should act as a title for the card. |
+| `component` | `CardBody` | Sets the base element for the sub-component (`div` element by default). |
```

Comment:
It's not clear from this description when/how the `component` prop has any affect on a11y.

Reply:
Yeah, this and the line below I'm trying to think of how to expand on. The most I could really think of was after looking at some articles that went into accessible Card-like components was essentially not wrapping the entire card in an `<a>` element (so in this case passing `component="a"` to CardBody or CardFooter). Though we would probably just then have to explicitly mention all the elements to be wary of passing in for these props...

I could add on, "... When customizing this prop, be sure to not set the base element of this sub-component as an interactive element such as a button or anchor". Or maybe just mention being sure to not nest interactive elements inside of interactive elements?

I'm also fine with removing these lines for now as well, though.

Reply:
If they nest interactive elements, it should flag an a11y error if they are doing any automated testing, so I think it might be okay to leave these off.